astropy/modeling/functional_models.py

```diff
@@ -1370,3 +1454,109 @@ def custom_model_1d(func, func_fit_deriv=None):
             "All parameters must be keyword arguments")
 
     return custom_model(func, fit_deriv=func_fit_deriv)
+
+
+class Sersic2D(Fittable2DModel):
+    r"""
+    Two dimensional Sersic surface brightness profile.
+
+    Parameters
+    ----------
+    amplitude : float
+        Central surface brightness, within r_eff. 
+    r_eff : float
+        Effective (half-light) radius
+    n : float
+        Sersic Index.
+    x_0 : float, optional
+        x position of the center.
+    y_0 : float, optional
+        y position of the center. 
+    ellip : float, optional
+        Ellipticity.
+    theta : float, optional
+        Rotation angle in radians, counterclockwise from
+        the positive x-axis. 
+
+    See Also
+    --------
+    Gaussian2D, Moffat2D, Lorentz2D
```

Comment:
The docs-build currently fails on travis-ci, because of the `Lorentz2D` reference, which doesn't exist. Just remove it...